src/olympia/lib/es/utils.py

```diff
@@ -36,7 +36,11 @@ def index_objects(ids, model, extract_func, index=None, transforms=None,
     if transforms is None:
         transforms = []
 
-    qs = objects.no_cache().filter(id__in=ids)
+    if hasattr(objects, 'no_cache'):
```

Comment:
where is this attribute set?

Reply:
It's the `no_cache()` method from `caching.base.CachingManager`. Models that use `UncachedManagerBase` and not `ManagerBase` therefore don't have it.

Reply:
👍